c/meterpreter/source/extensions/priv/namedpipe.c

```diff
@@ -390,3 +390,12 @@ DWORD elevate_via_service_namedpipe2(Remote * remote, Packet * packet)
 
 	return dwResult;
 }
+
+BOOL is_pipe_exists(LPWSTR pPipeName)
```

Comment:
Would you mind renaming this to `does_pipe_exist`? That would be a little easier to read.

Reply:
sure!

Reply:
This has not been resolved, naming is still `is_pipe_exists`.